signac/synced_collections/buffers/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""This subpackage defines the buffering protocol and supported implementations.
+
+No buffers are imported by default. Users should import desired buffers as
+needed.
+"""
+__all__ = [""]
```

Comment:
Should it be an empty list, or a list containing an empty string? Is this line needed? If unsure, I vote to delete.

Reply:
Same discussion as above.